ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -33,23 +34,43 @@ def __init__(self):
 
         rospy.Subscriber('/current_pose', PoseStamped, self.pose_cb)
         rospy.Subscriber('/base_waypoints', Lane, self.waypoints_cb)
+        rospy.Subscriber('/traffic_waypoint', Int32, self.traffic_cb)
+        rospy.Subscriber('/obstacle_waypoint', Lane, self.obstacle_cb)
 
-        # TODO: Add a subscriber for /traffic_waypoint and /obstacle_waypoint
-
-        self.final_waypoints_pub = rospy.Publisher('final_waypoints', Lane,
-                                                   queue_size=1)
+        self.pub = rospy.Publisher('final_waypoints', Lane, queue_size=1)
```

Comment:
I notice this was wrong from the beginning, but I think the topic name should be `/final_waypoints` (backslack in the beginning)

Reply:
Both seem to work, but I agree - adding the slash looks better :)